views/js/controllers/configuration/back.js

```diff
@@ -166,14 +167,22 @@ $(document).ready(function() {
         function() {
             let name = $(this).data('module_name');
             $('.module-list [data-module_name='+ name +']').addClass('active');
-            $(this).find('.on-element').removeClass('hide');
+            $(this).find('.on-element').addClass('hide');
```

Comment:
What do you think about `.active-element` to properly distinguish the hovered from the active panel?